### PR TITLE
arranged routes in order

### PR DESCRIPTION
**Pull request by tapansawant**
Arranged all the routes in alphabetical order.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-app.use('/pullrequests', require('./pullrequests.js'))
 app.use('/contributions', require('./contributions'))
-app.use('/badges', require('./badges.js'))
-app.use('/trade', require('./trading.js'))
+app.use('/healthcheck', require('./healthCheck.js'))
+app.use('/members', require('./members.js'))
+app.use('/pullrequests', require('./pullrequests.js'))
 app.use('/stocks', require('./stocks.js'))
-app.use('/auctions', require('./auctions.js'))
+app.use('/tasks', require('./tasks'))
```
Comment (swarajpure) [inline routes/index.js:13]: `contributions` and `tasks` are not having `.js` extensions, can you please add?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thanks for the change! 🎉 

Can you also make sure that all the route files are having `.js` extensions? It'll help maintain some form of consistency throughout the file.

Congratulations on your first pull request in Real Dev Squad! 🥳

File: routes/index.js (hunk 1 above)
Comment (tapansawant) [reply routes/index.js:13]: Sure I will do it

**Comment by swarajpure** [comment]
Closing in lieu of #404

**Pull request closed without merging**